src/main/java/adry/projectile/ProjectileFrame.java

```diff
@@ -110,34 +124,39 @@ public void actionPerformed(ActionEvent e) {
                         projectile.setSeconds(
```

Comment:
The only thing you should be doing in the actionPerformed method is calling `calculate()` like you do in the other Listeners.